.github/workflows/semver-checks-pr-label.yml

```diff
@@ -0,0 +1,83 @@
+on:   # yamllint disable-line rule:truthy
+  workflow_run:
+    workflows: [Check semver breaks]
+    types: [completed]
+
+name: Check semver breaks - Label and Comment PR
+
+permissions:
+  contents: read
+  pull-requests: write
```

Comment:
See https://github.com/storopoli/rust-bitcoin/actions/runs/9919877598 for the failure

Reply:
Just a note that if you add other jobs to this file, they'll all get this set of permissions by default.

If this is something you might want to avoid, you can move this block as-is to sit under the `jobs: -> Download:` key and then it'll apply to that one job only.

All the same to me, just wanted to offer an alternative for you to consider.

Reply:
We shouldn't add new jobs here but I wouldn't mind if it got moved.

Reply:
Done! Thanks!